<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bullsFilter</class>
 <widget class="QDialog" name="bullsFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Bull filter</string>
  </property>
  <widget class="QComboBox" name="NumberCombo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="NameCombo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="InvCombo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="MarkCombo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="BreedCombo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="LineCombo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="FamilyCombo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="BDateCombo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="BPlaceCombo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="InDateCombo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="ColourCombo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="OwnerCombo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QLabel" name="BreedLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Breed</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumberTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2147483647</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="BreedTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="NumberLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="InvNumTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2147483647</number>
   </property>
  </widget>
  <widget class="QLabel" name="LineLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </widget>
  <widget class="QLabel" name="InvNumLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>InvNumber</string>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MarkTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MarkLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OwnerTxt">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="OwnerLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Owner</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="InDateTxt">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="FamilyLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Family</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FamilyTxt">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="InDateLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>InDate</string>
   </property>
  </widget>
  <widget class="QLabel" name="ColourLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Colour</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BirthPlaceTxt">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="BirthDateLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BirthDate</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="BirthDateTxt">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ColourTxt">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="BirthPlaceLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BirthPlace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
